bitbots_blackboard/bitbots_blackboard/capsules/costmap_capsule.py

```diff
@@ -316,6 +304,13 @@ def get_gradient_direction_at_field_position(self, x: float, y: float):
         return math.atan2(grad[1], grad[0])
 
     def get_cost_of_kick_relative(self, x: float, y: float, direction: float, kick_length: float, angular_range: float):
+        """
+        Returns the cost of a kick at the given position and direction relative to the robot
+        :param x: Field coordiante in the x direction
```

Comment:
This is not in the field/map coordinate system. They are given in the base footprint frame aka relative to the robot.